patches/chrome-browser-buildflags.gni.patch

```diff
@@ -0,0 +1,16 @@
+diff --git a/chrome/browser/buildflags.gni b/chrome/browser/buildflags.gni
+index 653d675e736d55330ef86a62436f1d0c16406e60..fb014f4ea6882759cb2f78e11ece27fd01c60cf9 100644
+--- a/chrome/browser/buildflags.gni
++++ b/chrome/browser/buildflags.gni
+@@ -6,9 +6,9 @@ import("//build/config/chrome_build.gni")
+ 
+ declare_args() {
+   # By default, only branded builds integrate with automatic updates.
+-  enable_updater = is_chrome_branded
++  enable_updater = is_chrome_branded || is_official_build
```

Comment:
this is a declare_args section so we can set this in config.js

Reply:
Fixed in 7996e758b709a434e0808e83e0f6a18f07ea0948.